core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -146,7 +146,18 @@ public void importData() throws IOException, DaoException {
             {
                 String[] parts = line.split("\t", -1 ); // the -1 keeps trailing empty strings; see JavaDoc for String
                 MafRecord record = mafUtil.parseRecord(line);
+                CancerStudy cancerStudy = DaoCancerStudy.getCancerStudyByInternalId(geneticProfile.getCancerStudyId());
+                String genomeBuildName;
+                try {
+                    String referenceGenome = cancerStudy.getReferenceGenome();
+                    genomeBuildName = DaoReferenceGenome.getReferenceGenomeByGenomeName(referenceGenome).getBuildName();
+                } catch (NullPointerException e) {
+                    genomeBuildName = ReferenceGenome.HOMO_SAPIENS_DEFAULT_GENOME_BUILD;
+                }
 
+                if (!record.getNcbiBuild().equalsIgnoreCase(genomeBuildName)) {
+                    ProgressMonitor.setCurrentMessage("Genome Build Name does not match, expecting " + genomeBuildName);
```

Comment:
NCBI Build name currently is an option field for the maf file. Was agreed at meeting with JJ, Pieter, Benjamin, the default genome will be used if no build name supplied.
Will read the reference genome information from the portal.properties file not using a symbolic link.

Reply:
genome build names never be normalized (hg19, GRCh37, 37), so we only send out a warning messages not raise an exception or stop importing.

Reply:
There are 2 scenarios here: (1) if no build info, fall back to default: not ideal, but OK (I don't see it implemented though); (2) build info is wrong: please throw an error in this case.

Reply:
@pieterlukasse , it was agreed during meeting with @jjgao and @n1zea144, we only send out a warning message.

Reply:
Ok, given it is already strictly checked for `human` and `mouse` at https://github.com/cBioPortal/cbioportal/blob/83f4c918e12ec83121a1821a42044abcd33fe4ab/core/src/main/scripts/importer/validateData.py#L1686 (`checkNCBIbuild(self, value)`), resulting in validation error, I can live with no real check in this layer. Once the validation is extended to other species (perhaps rat) or restricted to only human and mouse, then this code can actually be removed from the java layer.
Only remark for now is: use `ProgressMonitor.logWarning` instead of `ProgressMonitor. setCurrentMessage`

Reply:
changed to logWarning.